### PR TITLE
File renames

### PR DESCRIPTION
**Pull request by mairaw**
Not sure why our migration tool not always respected the new files names we've defined. There are a few of those but the ones included in this PR are a little bit more problematic because they're either GUIDs or have special characters.

### CODE REVIEW

**Comment by mairaw** [comment]
This is ready for review. It would be good to get this one merged asap since it impacts the redirection I'm requesting from msdn to docs for .net docs. @rpetrusha @BillWagner @GuardRex

**Comment by GuardRex** [comment]
@mairaw I'll do it now.

File: xml/System.Windows.Forms/DataGridView.xml [hunk 1]
```diff
@@ -5624,7 +5624,7 @@
   
  For more information, see [Displaying Data in the Windows Forms DataGridView Control](~/docs/framework/winforms/controls/displaying-data-in-the-windows-forms-datagridview-control.md). The following table provides direct links to common tasks related to the <xref:System.Windows.Forms.DataGridView.DataSource%2A> property.  
   
- See [Walkthrough: Creating a Master/Detail Form Using Two Windows Forms DataGridView Controls](~/docs/framework/winforms/controls/c5fa29e8-47f7-4691-829b-0e697a691f36.md)and [How to: Bind Objects to Windows Forms DataGridView Controls](~/docs/framework/winforms/controls/how-to-bind-objects-to-windows-forms-datagridview-controls.md).  
+ See [Walkthrough: Creating a Master/Detail Form Using Two Windows Forms DataGridView Controls](~/docs/framework/winforms/controls/creating-a-master-detail-form-using-two-datagridviews.md)and [How to: Bind Objects to Windows Forms DataGridView Controls](~/docs/framework/winforms/controls/how-to-bind-objects-to-windows-forms-datagridview-controls.md).
```
Comment (GuardRex) [inline xml/System.Windows.Forms/DataGridView.xml]: Space before "and" right after the link.

File: docs/framework/xaml-services/xml-character-entities-and-xaml.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ XAML uses character entities defined in XML for special characters. This topic d
   
  The main exception is that braces ({ and }) have significance in XAML because these characters inform a XAML processor that a character sequence enclosed by braces must be interpreted as a markup extension. For more information about markup extensions, see [Markup Extensions for XAML Overview](../../../docs/framework/xaml-services/markup-extensions-for-xaml-overview.md).  
   
- However, you can still display the braces as literal characters by using an escape sequence that is particular to XAML instead of XML. For more information, see {} Escape Sequence - Markup Extension.  
+ However, you can still display the braces as literal characters by using an escape sequence that is particular to XAML instead of XML. For more information, see [{} Escape Sequence - Markup Extension](escape-sequence-markup-extension.md).
```
Comment (GuardRex) [inline docs/framework/xaml-services/xml-character-entities-and-xaml.md]: This one had a bit of a :boom: on the review site: https://review.docs.microsoft.com/en-us/dotnet-new/articles/framework/xaml-services/xml-character-entities-and-xaml?branch=pr-en-us-2223

**Review by guardrex — CHANGES_REQUESTED** [review]

File: docs/framework/xaml-services/xml-character-entities-and-xaml.md (hunk 2 above)
Comment (mairaw) [inline docs/framework/xaml-services/xml-character-entities-and-xaml.md]: Good catch. It's already like that in production. Usually it's a bad character in the metadata section like the ones i fixed for https://docs.microsoft.com/en-us/dotnet/articles/framework/xaml-services/%7B%7D-escape-sequence-markup-extension above

Comment (GuardRex) [inline docs/framework/xaml-services/xml-character-entities-and-xaml.md]: Sprinkled in there by the looks of it ...

capture

Comment (GuardRex) [inline docs/framework/xaml-services/xml-character-entities-and-xaml.md]: `0A` = LF ... that's not it. I'm curious tho if it can be spotted.

Comment (GuardRex) [inline docs/framework/xaml-services/xml-character-entities-and-xaml.md]: Single quotes???

capture

**Review by guardrex — APPROVED** [review]

**Pull request merged**